apps/rush/src/utilities/PackageReviewChecker.ts

```diff
@@ -30,7 +27,8 @@ export default class PackageReviewChecker {
   }
 
   public saveFile(): void {
-    this._packageReviewConfiguration.saveFile(this._rushConfiguration.packageReviewFile);
+    this._rushConfiguration.browserApprovedPackages.saveToFile();
```

Comment:
I avoided doing this because I was feeling lazy.  But yeah, it would make the code more readable.  I made that change.

---
In reply to: [116122068](https://github.com/Microsoft/web-build-tools/pull/181#discussion_r116122068) [](ancestors = 116122068)